crates/resolver-tests/README.md

```diff
@@ -0,0 +1,13 @@
+# resolver-tests
+
+## The aim
+
+This crate aims to test the resolution of cargo's resolver. It implements a [SAT solver](https://en.wikipedia.org/wiki/SAT_solver) to compare resolution of cargo's resolver.    
+This ensures that cargo's dependency resolution is valid on the [SAT](https://en.wikipedia.org/wiki/Boolean_satisfiability_problem).   
+
+## About the test
+
+The cargo's resolver is very sensitive to what order it tries to evaluate constraints.  This makes it incredibly difficult     
```

Comment:
> The cargo's resolver

Nit: "The resolver in cargo" or "Cargo's resolver"

Reply:
`Cargo's resolver` seems more succinct